src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my
+          ${sex === 'm'
+            ? 'wife'
+            : 'husband'}`
+          : 'I am not married'}
```

Comment:
move this logic out of jsx, it's too complicated and hard to read. do not afraif even create 2 variables such status, marriedString